scalding-core/src/test/scala/com/twitter/scalding/StringUtilityTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.twitter.scalding
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class StringUtilityTest extends WordSpec with Matchers {
```

Comment:
wait, why?
I think we should assert that this matches String.split, or `String.split(s, -1)` if you want that behavior. But we have a reference that we know works.